library/swift/stats/Stats.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+struct Stats {
+  /// Convert tags.
+ static func convert(tags: [Tag]) -> [String: String] {
+    var tagMap = [String: String]()
+    for tag in tags {
+        tagMap[tag.key] = tag.value
+    }
+    return tagMap
+  }
```

Comment:
For swift, the more common way is to make a static function (as oppose to having a method in an interface like kotlin), and for the parity, this function is namespaced as `Stats` - same to the kotlin `Stats` interface